source/api/presentation/3.0/index.md

```diff
@@ -579,7 +579,7 @@ The value _MUST_ be an array of JSON objects. Each object _MUST_ have the `id` a
 ```
 
 ##### seeAlso
-A link to a machine readable document that is related to the resource with the `seeAlso` property, such as an XML or RDF description. Properties of the document should be given to help the client select between multiple descriptions (if provided), and to make appropriate use of the document. If the relationship between the resource and the document needs to be more specific, then the document should include that relationship rather than the IIIF resource. Other IIIF resources, such as a related Manifest, are valid targets for `seeAlso`.
+A link to a machine readable document that is related to the resource with the `seeAlso` property, such as an XML or RDF description. Properties of the document should be given to help the client select between multiple descriptions (if provided), and to make appropriate use of the document. If the relationship between the resource and the document needs to be more specific, then the document should include that relationship rather than the IIIF resource. Other IIIF resources, such as a related Manifest, are valid targets for `seeAlso`. The URI of the document _MUST_ identify a single representation with a single `format`. For example, if the same data exists in JSON and XML, with a URL that supports content negotiation for the two formats, then the URIs of each representation can be provided separately. 
```

Comment:
> ... JSON and XML, with a URL that ...

URL or URI?

Reply:
> then the URIs of each representation can be provided separately. 

Also, not sure this makes sense to me. We're saying that there should be a JSON object per `format`, even if some those formats have the same URI and require conneg to get the desired representation/serialization, is that correct?

Reply:
@zimeon will suggest a rewrite of last two sentences.

Reply:
Urgh. Technically they're locators, but we should stick with just URI.

The URI to use is the one for the representation, not the negotiable resource. It's hard to express in generic terms without requiring a deep knowledge of HTTP conneg details :(

Reply:
I suggest replacing the last sentence with:

> For example, if the same data exists in JSON and XML, then separate JSON objects should be added for each representation, with distinct `id` and `format` properties. 

if we want to be explicit about conneg then we could also add:

> Clients are not expected to understand how to content negotiate based on the `format` property of `seeAlso` resources.

but I'm include to go with just the "For example..." sentence.

Reply:
:+1: